.bmad-core/templates/story-tmpl.yaml

```
@@ -0,0 +1,137 @@
+  version: 2.0
+  output:
+    format: markdown
+    filename: docs/stories/{{epic_num}}.{{story_num}}.{{story_title_short}}.md
+    title: "Story {{epic_num}}.{{story_num}}: {{story_title_short}}"
+
+workflow:
+  mode: interactive
+  elicitation: advanced-elicitation
+
+agent_config:
+  editable_sections: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)